src/store/store.ts

```diff
@@ -979,15 +977,15 @@ export const useStore = defineStore('store', {
 
     /** Whether NAICS data has changed. */
     hasNaicsChanged (): boolean {
-      const currentNaicsCode = this.getBusinessInformation?.naicsCode
+      const currentNaicsCode = this.getBusinessInformation.naicsCode
```

Comment:
Business Information should always exist (and it's initialized as an object, no null), so conditional chaining is not needed (and may hide real errors during development).